### PR TITLE
LDPlayer.LDPlayer version 9.0.36

### PR DESCRIPTION
**Pull request by TaQuangKhoi**
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95662)

### CODE REVIEW

**Comment by TaQuangKhoi** [comment]
@microsoft-github-policy-service agree

File: manifests/l/LDPlayer/LDPlayer/9.0.36/LDPlayer.LDPlayer.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: LDPlayer.LDPlayer
+PackageVersion: 9.0.36
+PackageLocale: en-US
+Publisher: LDPlayer
+PackageName: LDPlayer
```
Comment (stephengillie) [inline manifests/l/LDPlayer/LDPlayer/9.0.36/LDPlayer.LDPlayer.locale.en-US.yaml:8]:
```suggestion
PackageName: LDPlayer
PackageUrl: https://www.ldplayer.net/
```

**Comment by stephengillie** [comment]
Hi @TaQuangKhoi,

Does the installer have an unattended mode? Currently the installer requires user interaction. 
image

**Comment by TaQuangKhoi** [comment]
Hi @stephengillie ,

I think that the installer doesn't have unattended mode.

Is it required to have this mode to commit to repo?

**Comment by stephengillie** [comment]
> Hi @stephengillie ,
> 
> I think that the installer doesn't have unattended mode.
> 
> Is it required to have this mode to commit to repo?

Hi @TaQuangKhoi,

Unfortunately, installers must have a fully attended mode to be accepted into the Winget public manifest at this time. This installer might be supported with: 
* https://github.com/microsoft/winget-cli/issues/910

**Pull request closed without merging**